### ISSUE TITLE
Post processing to get voxels

### ISSUE THREAD

**Issue by tstandley** [fanhqme/PointSetGeneration#1]
Hi!

In the paper you mentioned a model that you train to post process your point set to get a voxelized representation which you use to compare to R2N2. 

For my use case I need voxels. Is there any chance you could release this model/code as well?

Thanks!

**Comment by Jerrypiglet** [comment]
Same request here. I am working on similar projects and it would be nice if I can get your data and evaluate on them. Thanks a lot!

**Comment by ChenFengYe** [comment]
Same request! Hope to get your reply！

**Comment by HaolongZ** [comment]
Yep. Need that you. Anyone have the mean to do the conversion to voxel please share with me, Thanks!

**Comment by xiaomingjie** [comment]
Same request to calculate IoU. Hope to get your help!

**Comment by guoyan1991** [comment]
@seanmj @tstandley @Jerrypiglet @ChenFengYe @HaolongZ 
Hello， I have finished the process of calculating voxel IoU. I don't retrain the network , And I use the network author provided corresponding to 3d - R2N2 network dataset. However, the results provided in the article are not reachable. Maybe I have some problems with data processing. If you are studying relevant content, I hope to communicate with you, and I hope to get your reply sincerely.

**Comment by HaoxuZhang** [comment]
Same request

**Comment by HaoxuZhang** [comment]
@guoyan1991 
Hello, could you please show the code of the process of calculating voxel IoU? Hope to get your help!

**Comment by guoyan1991** [comment]
@HaoxuZhang I use traditional algorithm transformed the point cloud into a 32,32,32 voxel, and then use a 3DCNN described in the article. But I didn't get the results in the paper, it is just a little bit better than the results of 3D-R2N2.

**Comment by guoyan1991** [comment]
@HaoxuZhang I used the decoding part network of 3d-r2n2 to calculate IOU.

**Comment by ypzuo** [comment]
@guoyan1991 hi, I'm new in this field. Can you tell me the steps and the related code in detail? Thank you very much !
